draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -7592,14 +7592,16 @@ If-Unmodified-Since: Sat, 29 Oct 1994 19:43:31 GMT
      <x:ref>Range</x:ref> and <x:ref>If-Range</x:ref> are present,
      evaluate the <x:ref>If-Range</x:ref> precondition:</t>
      <ul>
-       <li>if the validator matches and the Range specification is
-           applicable to the <x:ref>selected representation</x:ref>, respond
-           <x:ref>206 (Partial Content)</x:ref></li>
+       <li>if true and the Range is
+           applicable to the <x:ref>selected representation</x:ref>,
+           respond <x:ref>206 (Partial Content)</x:ref></li>
+       <li>otherwise, ignore the Range header field and
+           respond <x:ref>200 (OK)</x:ref></li>
```

Comment:
```suggestion
       <li>if true and the <x:ref>Range</x:ref> is
           applicable to the <x:ref>selected representation</x:ref>,
           respond <x:ref>206 (Partial Content)</x:ref></li>
       <li>otherwise, ignore the <x:ref>Range</x:ref> header field and
           respond <x:ref>200 (OK)</x:ref></li>
```